lib/rpc/tls.go

```diff
@@ -155,6 +156,11 @@ func CredentialsFromPackage(
 	if err != nil {
 		return nil, nil, trace.Wrap(err)
 	}
+	now := time.Now()
+	err = validateCerts(tlsArchive, now)
```

Comment:
Hm, I'm not entirely sure about this - the purpose of this function is just to load the certs from the specified package, valid or not, so adding validation here may be mixing concerns. Or is it hard to add validation for what this method returns (TransportCredentials)?

Also, nit: not sure why a separate variable for "now" is needed.